src/pages/gateway/source-handlers.md

```diff
@@ -24,11 +24,11 @@ Only alphanumerical characters are allowed in source handler names.
 
 ## OpenAPI
 
-The [OpenAPI] handler allows you to connect to an OpenAPI-complaint REST service endpoint or static Swagger schemas using a `.json` or `.yaml` file.
+The [OpenAPI] handler allows you to connect to an OpenAPI-compliant REST service endpoint or static Swagger schemas using a `.json` or `.yaml` file.
 
 <InlineAlert variant="info" slots="text"/>
 
-When using a Swagger schema, API Mesh can only access `application/json` content from the Swagger API definition.
+When using a Swagger schema, API Mesh can only access `application/json` content from the Swagger API definition. API Mesh does not accept a `/` or a wildcard (`**`) as a content type.
```

Comment:
@jhadobe the content type customers using wrongly is `*/*`.  we can rewrite it the passage like this
API Mesh does not accept a wildcard (`*/*`) as a content type.